tests/IdempotencyTokenMiddlewareTest.php

```diff
@@ -3,7 +3,6 @@
 
 use Aws\IdempotencyTokenMiddleware;
 use Aws\Result;
-use GuzzleHttp\Promise;
```

Comment:
It looks like this could was meant to alleviate the fully qualified namespace on [line 26](https://github.com/aws/aws-sdk-php/pull/1453/files#diff-5026ea339e2c8e0c4c6c0e0bbc7c3167R26).

This could be `return Promise\promise_for(new Result([]));` instead.